tests/test_folium.py

```diff
@@ -114,40 +114,48 @@ def test_init(self):
                     'name': 'TileLayer',
                     'id': '00000000000000000000000000000000',
                     'children': {}
-                    }
                 }
             }
+        }
 
     def test_builtin_tile(self):
         """Test custom maptiles."""
 
-        default_tiles = ['OpenStreetMap', 'Stamen Terrain', 'Stamen Toner']
+        default_tiles = [
+            "OpenStreetMap",
+            "Stamen Terrain",
+            "Stamen Toner",
+            "Mapbox Bright",
+            "Mapbox Control Room",
+            "CartoDB positron",
+            "CartoDB dark_matter",
+        ]
         for tiles in default_tiles:
             m = folium.Map(location=[45.5236, -122.6750], tiles=tiles)
-            tiles = ''.join(tiles.lower().strip().split())
-            url = 'tiles/{}/tiles.txt'.format
-            attr = 'tiles/{}/attr.txt'.format
-            url = m._env.get_template(url(tiles)).render()
-            attr = m._env.get_template(attr(tiles)).render()
+            tiles = "".join(tiles.lower().strip().split())
+            url = f"tiles/{tiles}/tiles.txt"
```

Comment:
f-strings are great, but folium still supports Python 3.5. Right now we don't run tests for Python 3.5, but that's something we should probably do. Can you use normal `format()` instead?

Reply:
Ok @Conengmo , sorry for that, it is corrected now. 
One question: I wanted to add also a test for tiles needing an API key (cloudmade and mapbox), but I noticed that you suppressed one in your previous commit (`test_cloudmade`). Any reason for that ?

Reply:
@Conengmo Ah, I see, you removed the test `test_cloudmade` following GH-1127.